.vscode/launch.json

```diff
@@ -30,14 +30,15 @@
 				"--extensionDevelopmentPath=${workspaceFolder}"
 			],
 			"env": {
+				"AZCODE_DOCKER_IGNORE_BUNDLE": "",
```

Comment:
It may have been this bug: https://github.com/microsoft/vscode/issues/100265

But it seems like env vars that were set in a previous launch config persisted into the next launch, which was messing a lot of things up.

Reply:
Yes, I've seen this happen from time to time.